base-2016/readme.md

```diff
@@ -145,6 +145,39 @@ As you can see, we can still use 'include' for small blocks of HTML, even though
 we're using template inheritance. This way we can keep our themes very
 structured and organized.
 
+### Showing all fields without defining them
+
+Because this is a general purpose theme, we try to make it work without problems
+for any defined contenttype. This means we'll need to render all available
```

Comment:
Ninja branding fixes incoming … you really need to keep a closer eye on :koala:s when you're typing, thos sneaking little rascals will take over your keyboard when you least expect it and … **this** :wink: